### PR TITLE
Feature/real api responses

### PR DESCRIPTION
**Pull request by christophermlne**

### CODE REVIEW

**Comment by ghost** [comment]
Hi Christopher.
I'm a flight instructor switching back to my old web des/dev job and updating my skills with JS, APIs, databases, etc... with a Bootcamp training program.
We're currently getting into APIs that I already know a bit about, and since we'll have to apply some knowledge in a little personal demo, I was looking for a TAF/METAR API to work with. (I've also started working on a personal project that will involve some aviation-oriented APIs, including TAFs/METARs). It made sense to me to check if NAV CANADA's APIs were available and I stumbled upon yours...
So my question is  "is yours available for public use and if so, how should I proceed to retrieve the info?"
Hope my request is not too clumsy...   Looking forward to hearing from you.
Have a great day
Hugo.

**Comment by christophermlne** [comment]
Hi Hugo,

Nav Canada does not make any api's available for this (at least, they didn't when i originally wrote this code). My first attempt was a web scraper and worked for a while until I got banned.
For the second attempt, I found the NOAA has the same data (i think Nav Canada pulls their data from the NOAA). However, after a week or so they also banned my IP. It was at that point that I gave up on this app.

The TL;DR is there is no reliable way that I could find to pull the necessary data. There is a mobile app called Avia Weather that is somehow able to do it so perhaps follow up with the author?

I have no issue if you want to fork the code in this repo. I have no plans to maintain or continue working on it.

**Comment by ghost** [comment]
Thanks a lot for your answer...
